tikv-configuration-file.md

```diff
@@ -1933,6 +1933,20 @@ Configuration items related to Raft Engine.
 + If there are multiple disks on your machine, it is recommended to store the data of Raft Engine on a different disk to improve TiKV performance.
 + Default value: `""`
 
+### `spill-dir` <span class="version-mark">New in v8.4.0</span>
+
++ The auxiliary directory for storing raft log files. When the disk where the `dir` specified is full, new raft logs will be stored under this directory. If this directory does not exist after configuration, it will be created when TiKV is started.
++ If this configuration is not set, it means that the auxiliary directory is disabled.
+
+> **Note:**
+>
+> - This configuration is only valid when the `dir` and `spill-dir` of the Raft Engine are specified as different disk drives.
+> - After enabling this feature, if you want to disable it, you need to perform the following operations before restarting TiKV:
```

Comment:
```suggestion
> - After enabling this feature, if you want to disable it, you need to perform the following operations before restarting TiKV; otherwise, TiKV won't be able to start:
```